### ISSUE TITLE
Allow pixel size to be set, manually or automatically

### ISSUE THREAD

**Issue by jmuhlich** [labsyspharm/minerva-author#55]
We should read the pixel size (OME `Pixels.PhysicalSizeX` and `...Y`) from the input ome-tiff and place it in an editable field on the Sample Info tab. This should be presented as pixel size in microns in the UI -- microns is a reasonable scale for our target audience. This value should be written (inverted of course)  into the published exhibit.json under the `PixelsPerMicron` key to configure the scale bar. Well-written ome-tiffs will then automatically get a scale bar in their stories, and users can manually edit or override that value if it's incorrect or missing entirely.

**Comment by jmuhlich** [comment]
Let's bump this to high priority! We apparently have a lot of stories going out without a scale bar.

**Comment by jmuhlich** [comment]
How to extract the pixel size in microns:
```python
ome.images[0].pixels.physical_size_x_quantity.to('um').m```

**Comment by thejohnhoffer** [comment]
Resolved in Minerva Author v1.14.0
